### PR TITLE
Heap Sort

### PR DESCRIPTION
**Pull request by rajraii**
<hr>

## Description 📜

This documentation is about Heap sort covering all aspect of  heap sort algorithm # <3690> 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [X] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr
[HEAP SORT.md](https://github.com/girlscript/winter-of-contributing/files/7262369/HEAP.SORT.md)
>

### CODE REVIEW

**Comment by apeksha235** [comment]
@rajraii 
Add a file.
image

**Comment by rajraii** [comment]
> @rajraii 
> Add a file.
> image
> 

@apeksha235 done, can you verify it

**Comment by vasu-1** [comment]
> > @rajraii
> > Add a file.
> > image
> 
> @apeksha235 done, can you verify it

No it is not there !

**Comment by vasu-1** [comment]
**Add file and do commit !**
I can't see any commit after the suggestion of apeksha !!!

**Comment by vasu-1** [comment]
Also don't forget to push

**Comment by rajraii** [comment]
@vasu-1 okay done please verify it

File: C_CPP/Algorithmic Approaches/Sorting/HeapSort.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Heapify is a function used to convert a complete binary tree into max heap.
+-Heapify works in 3 steps
+  1. Check if the parent-node value is greater than child-node.
+  2. If the parent node has value lower than child node, Swap the parent node with child node.
+  3. Check for second child node.
+  
+## CODE FOR HEAP SORT 
+```
+#include <iostream>
+using namespace std;
```
Comment (vasu-1) [inline C_CPP/Algorithmic Approaches/Sorting/HeapSort.md:26]:
```suggestion
## CODE FOR HEAP SORT 
```c++
#include <iostream>
using namespace std;
```

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Just add this `c++` in your code..
Then your pr will be good to go for merge

**Comment by rajraii** [comment]
@vasu-1 okay ill do that asap

**Comment by vasu-1** [comment]
> @vasu-1 okay ill do that asap

Please sorry for this !!!

**Comment by rajraii** [comment]
@vasu-1 please verify it

**Comment by vasu-1** [comment]
#3690 
- see suvraneel sir has closed your Pr

**Pull request closed without merging**